src/extension/prompts/node/agent/openai/defaultOpenAIPrompt.tsx

```diff
@@ -99,15 +100,15 @@ export class DefaultOpenAIAgentPrompt extends PromptElement<DefaultAgentPromptPr
 			{this.props.availableTools && <McpToolInstructions tools={this.props.availableTools} />}
 			<NotebookInstructions {...this.props} />
 			<Tag name='outputFormatting'>
-				Use proper Markdown formatting in your answers. When referring to a filename or symbol in the user's workspace, wrap it in backticks.<br />
+				Use proper Markdown formatting. Backtick code identifiers (classes, functions, variables, commands). For file paths or specific code locations, do NOT use backticks—convert them to markdown links with line anchors; file path and line anchor link rules are defined in `FileLinkificationInstructions` below. Avoid duplicating those examples here.<br />
 				<Tag name='example'>
```

Comment:
Should the `FileLinkificationInstructions` be included in the output formatting section similar to `MathIntegrationRules`? Wondering we can avoid the `File path link examples live in fileLinkification section below.` section